xapian-applications/omega/templates/query

```diff
@@ -86,7 +87,6 @@ $if{$not{$msizeexact},
 $list{$map{$queryterms,$list{$html{$uniq{$unstem{$_}}},<b>,</b>/<b>,</b>}:&nbsp;$nice{$freq{$_}}},Term frequencies: ,$. ,}
 <br><small>Search took $time seconds</small>
 <table>
-$if{$opt{clicklog},$set{queryid,$hash{md5,$transform{s/^\s+|\s+$$//g,\1,$query}}}}
```

Comment:
Because for queries leading to zero hits (e.g. lorem ipsum) `QUERYID` was not being logged (or logged blank ) in `search.log`.